segway/_util.py

```diff
@@ -346,10 +350,9 @@ def extract_superlabel(label):
     sublabel part, in which case only the superlabel part is
     returned
     """
-    if isinstance(label, str):
-        return int(label.split(".")[0])
-    else:
-        return label
+    if '.' in label:
+        return label.split(".")[0]
+    return label
 
 
 def find_segment_starts(data, output_label):
```

Comment:
Could you elaborate why this function changed at all? Why did `find_segment_starts` change based on the label type changing? Some additional comments on what the code is going below with the data arrays would be greatly appreciated as well.

Reply:
`extract_superlabel` changed because it previously used the criteria of the label being a string to find `"superlabel.sublabel"`-formatted labels. However, all labels are now strings, so a new criteria (containing `'.'`) is needed. I haven't fully tested that works for superlabels.

`find_segment_starts` previously used `np.diff` to find where labels changed, as these are non-zero values. This cannot be applied to string arrays, so the code uses a very similar method to create a similar array of non-zero values at label changes.

Comments will be added.